test/remote/gateways/remote_checkout_test.rb

```diff
@@ -27,13 +27,14 @@ def test_successful_purchase
   end
 
   def test_successful_purchase_with_extra_options
-    response = @gateway.purchase(100, @credit_card, @options.merge(
+    options = @options.merge(
       currency: 'EUR',
       email: 'bob@example.com',
       order_id: generate_unique_id,
       customer: generate_unique_id,
       ip: '127.0.0.1'
-    ))
+    )
+    response = @gateway.purchase(100, @credit_card, options)
```

Comment:
This is an older version of a gateway that we no longer appear to support. I received 14 `404` errors on both my branch and master.